### DISCUSSION TITLE
How can I restore my account? Who can help me? 😭

### DISCUSSION THREAD

**Discussion by ShiQianzhi** [community/community#23546] [New to GitHub]
<p>I lost my 2fa verification code and 2fa recovery code, and lost my SSH key, but this email is really mine. Who can tell me how to recover my account? I’ve tried many methods, including data recovery. I’m going crazy. Who can help me :sob::sob::sob::sob::sob::sob::sob::sob::sob::sob::sob::sob:</p>

**Comment by nethgato** [comment, marked as answer]
<p>For accounts related questions like this, we’ll need to get you in touch with our Support team:</p>
<p><a href="https://support.github.com/contact" class="onebox" target="_blank" rel="noopener">https://support.github.com/contact</a></p>
<p>Please contact them directly, and should be able to assist.</p>

**Reply by Praveedsharoon** [reply]
packermoscow@gmail.com

No mobile number OTP two step verification my mobile phone is missing

**Reply by Praveedsharoon** [reply]
I want to recovery my this account packermoscow@gmail.com

**Comment by harshs2807** [comment]
<p><a class="mention" href="/shiqianz">@shiqianz</a> have you got any reply from support team as, same thing happens with me</p>

**Comment by MiguelIsaza95** [comment]
<p>The same is happening with my other account, and  I need to recover it because is the account that I created with a business email :frowning:</p>

**Comment by MuhammadBilalYar01** [comment]
<p>Same thing is happening with me but I am still able to commit code through my visual studio, only issue i am unable to access my account from browser. I have no idea how to recover my account.</p>

**Comment by MuhammadBilalYar01** [comment]
<p>Also GitHub support is very low as i receive first response after after and now after 3 days they is no response.</p>
